### PR TITLE
#3232 R Tools Default Working Directory 

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
image

### CODE REVIEW

File: src/Host/Client/Impl/Session/RSessionProvider.cs [hunk 1]
```diff
@@ -24,6 +24,7 @@ public sealed class RSessionProvider : IRSessionProvider {
         private readonly BrokerClientProxy _brokerProxy;
         private readonly ICoreServices _services;
         private readonly IConsole _console;
+        private readonly bool _workingDirectory;
```
Comment (int19h) [inline src/Host/Client/Impl/Session/RSessionProvider.cs]: Unused?

**Comment by int19h** [comment]
I might be confused, but I can't see the code that actually uses the new `WorkingDirectory` option?

**Comment by MikhailArkhipov** [comment]
The field was already in settings we just never exposed it. It is used in `AfterHostStarted`

File: src/Host/Client/Impl/Session/RSessionProvider.cs (hunk 1 above)
Comment (MikhailArkhipov) [inline src/Host/Client/Impl/Session/RSessionProvider.cs]: Yep

**Comment by int19h** [comment]
👍

**Comment by smortaz** [comment]
can i specify the initial directory to be where the project is created?

ie: [$Project_dir                       ]

(or whatever it is called)

**Comment by MikhailArkhipov** [comment]
No, no macros.

**Pull request merged**